src/components/Map/Map.js

```diff
@@ -0,0 +1,23 @@
+import './Map.css';
+import { useEffect } from 'react';
+import { mapService } from '../../services/mapService';
+
+const Map = () => {
+  useEffect(() => {
+    window.addEventListener('load', () => {
+      mapService.initMap();
+    });
+
+    return () => {
+      window.removeEventListener('load', () => {
```

Comment:
Are you sure this is going to work? I mean, what should be done in order to remove a given callback from an event?